### PR TITLE
UKRIW-367 - added support for disabled Preset

### PR DESCRIPTION
**Pull request by KatarzynaPaszko**
- Added support for disabled preset
- added info about Max size of AOI in Area Node
- removed Import/Export from Workflow tab footer

### CODE REVIEW

**Review by mlu-ssoft — CHANGES_REQUESTED** [review]

File: libs/map/feature-action-creator-panel/src/lib/content/workflow/nodes/area/value-node.component.tsx [hunk 1]
```diff
@@ -52,26 +46,40 @@ const formatArea = function (text: string, value: TAreaNode['value'], unit: 'km'
...
+export const ValueNode = ({ node, onClearButtonClick, error }: TValueNodeProps) => {
   const { t } = useTranslation();
   const { canActivateNode, isLast } = useActionCreator();
 
   return (
     <Node
       type={node.type}
       active={true}
-      text={formatArea(t('MAP.ACTION_CREATOR_PANEL.WORKFLOW.NODE.AREA.DESCRIPTION'), node.value, 'miles')}
+      text={formatArea(t('MAP.ACTION_CREATOR_PANEL.WORKFLOW.NODE.AREA.DESCRIPTION'), node.value, 'km')}
```
Comment (mlu-ssoft) [inline libs/map/feature-action-creator-panel/src/lib/content/workflow/nodes/area/value-node.component.tsx:78]: use unit from settings

**Review by mlu-ssoft — APPROVED** [review]

File: @types/i18next/resources.d.ts [hunk 2]
```diff
@@ -149,7 +149,7 @@ interface Resources {
             AREA: {
               TITLE: 'Area';
               DEFAULT_VALUE: 'Drawing 1';
-              INSTRUCTIONS: 'Use the drawing tools to define an area of interest';
+              INSTRUCTIONS: 'Use the drawing tools to define an area of interest. Please note we cannot return results for areas greater than 1,000m2';
```
Comment (mlu-ssoft) [inline @types/i18next/resources.d.ts:152]: generate this file once again, I think this string is wrong

File: libs/map/feature-action-creator-panel/src/lib/content/workflow/nodes/area/value-node.component.tsx [hunk 3]
```diff
@@ -63,15 +75,34 @@ type TValueNodeProps = {
...
+  const { measurmentUnit, aoiLimit } = useSettings();
+  const aoiLimitInfo = useMemo(
+    () =>
+      t('MAP.ACTION_CREATOR_PANEL.WORKFLOW.NODE.AREA.ALLOWED_SIZE', {
+        maxSize: convertUnits(aoiLimit, measurmentUnit),
+      }),
+    [t, aoiLimit, measurmentUnit]
+  );
+
+  const getErrorMessage = useCallback(() => {
```
Comment (mlu-ssoft) [inline libs/map/feature-action-creator-panel/src/lib/content/workflow/nodes/area/value-node.component.tsx:87]: why not `useMemo`?

**Review by mlu-ssoft — CHANGES_REQUESTED** [review]

File: libs/map/feature-action-creator-panel/src/lib/content/workflow/nodes/area/value-node.component.tsx (hunk 3 above)
Comment (KatarzynaPaszko) [reply libs/map/feature-action-creator-panel/src/lib/content/workflow/nodes/area/value-node.component.tsx]: updated

File: @types/i18next/resources.d.ts (hunk 2 above)
Comment (KatarzynaPaszko) [reply @types/i18next/resources.d.ts]: updated

**Review by mlu-ssoft — APPROVED** [review]

**Pull request merged**